services/refreshToken.test.js

```diff
@@ -0,0 +1,84 @@
+import authenticationAdapter from 'adapters/Authentication';
+import * as authenticationHelper from 'helpers/authentication';
+import { mockAxiosError } from 'tests/error';
+
+import { refreshToken, refreshTokenErrorPath } from './refreshToken';
+
+jest.mock('adapters/Authentication');
+
+describe('refreshToken', () => {
+  const mockTokenType = 'token type';
+  const mockAccessToken = 'access token';
+  const mockRefreshToken = 'refresh token';
+  const successResponse = {
+    data: {
+      type: 'authorization',
+      attributes: {
+        tokenType: mockTokenType,
+        accessToken: mockAccessToken,
+        refreshToken: mockRefreshToken,
+      },
+    },
+  };
+  const mockInvalidError = mockAxiosError(422);
+
+  describe('given the refresh token is valid', () => {
+    it('sets the tokens', async () => {
+      const authenticationAdapterSpy = jest.spyOn(authenticationAdapter, 'refreshToken');
+      authenticationAdapterSpy.mockResolvedValue(successResponse);
```

Comment:
How about merging these lines?

```suggestion
      const authenticationAdapterSpy = jest.spyOn(authenticationAdapter, 'refreshToken').mockResolvedValue(successResponse);
```

Reply:
done in 191115e3b3c247e8f417ba60f0c6ca950251c98d